dpctl/tensor/_ctors.py

```diff
@@ -106,6 +90,21 @@ def _array_info_sequence(li):
     return (n,) + dim, dt, device
 
 
+def _c_contig_strides(sh):
```

Comment:
Is this meant to be used somewhere or is it a leftover from something?

Reply:
I am going to remove it. I was used in the initial variant of the fix which was replaced with the call to `_empty_like_orderK`.

Reply:
I removed the code and force-pushed the changes.